index.html

```diff
@@ -3600,94 +3733,86 @@ <h2>ExposedThing Examples</h2>
     </section> <!-- ExposedThing Examples -->
   </section> <!-- ExposedThing -->
 
-  <section data-cite="webidl" data-dfn-for="ThingDiscovery">
-    <h2>The <dfn>ThingDiscovery</dfn> interface</h2>
+  <section data-cite="webidl" data-dfn-for="ThingDiscoveryProcess">
+    <h2>The <dfn>ThingDiscoveryProcess</dfn> interface</h2>
     <p>
       Discovery is a distributed application that requires provisioning and support from participating network nodes (clients, servers, directory services). This API models the client side of typical discovery schemes supported by various IoT deployments.
     </p>
     <p>
-      The {{ThingDiscovery}} object is constructed given a filter and provides the properties and methods controlling the discovery process.
+      The {{ThingDiscoveryProcess}} object provides the properties and methods
+      controlling the discovery process and returning the results.
     </p>
     <pre class="idl">
       [SecureContext, Exposed=(Window,Worker)]
-      interface ThingDiscovery {
+      interface ThingDiscoveryProcess {
         constructor(optional ThingFilter filter = null);
-        readonly attribute ThingFilter? filter;
-        readonly attribute boolean active;
         readonly attribute boolean done;
         readonly attribute Error? error;
-        undefined start();
-        Promise&lt;ThingDescription&gt; next();
         undefined stop();
         async iterable&lt;ThingDescription&gt;;
```

Comment:
Nit: double semicolon

Reply:
Old error, not in this PR, but will fix with other things if I need something else fixing, too.

Reply:
Ahh, I see 🙈